app/js/handlebarsScoreTable.ts

```diff
@@ -411,8 +411,20 @@ function addEventListenersForMoreInfoButtons() {
             document.querySelector('tr[data-id="' + userId + '"]').classList.toggle('hide')
             if(this.textContent == 'More info') {
                 this.textContent = 'Less info';
+                e.target.parentElement.parentElement.style.background = "#fff";
+                let percentageElems = document.querySelectorAll('tr[data-percentage]')
+                percentageElems.forEach(function (percentageElem) {
+                    if (Number(percentageElem.getAttribute("data-percentage")) >= 97) {
+                        percentageElem.classList.add('top-grade')
+                    } else if (Number(percentageElem.getAttribute("data-percentage")) >= 70) {
+                        percentageElem.classList.add('passing-grade')
+                    } else {
+                        percentageElem.classList.add('dubious-grade')
+                    }
+                })
             } else {
                 this.textContent = 'More info';
+                e.target.parentElement.parentElement.removeAttribute("style");
```

Comment:
related to the previous comment, this should be removing a css class instead of removing the style attribute. this is also a good idea because someone in the future might use the style attribute for something unrelated and then wonder why it suddenly disappears.

Reply:
all done :)